### PR TITLE
I created a test to test the implementation of a ValueError in the cr…

### PR DESCRIPTION
**Pull request by RafaelPignata622**
…eation of masks for negative energies in the jcirc method

### CODE REVIEW

**Comment by RafaelPignata622** [comment]
This merge is not going to happend now ... i need to fix and complete some things

File: galaxychop/core.py [hunk 1]
```diff
@@ -491,6 +491,12 @@ def jcirc(self, bin0=0.05, bin1=0.005):
         (neg,) = np.where(E_tot <= 0.0)
         (neg_star,) = np.where(Etot_s <= 0.0)
 
+        if len(neg) == 0:
+            raise ValueError("All Energy values are positive")
```
Comment (ornemarioni) [inline galaxychop/core.py:495]: I would put the whole sentence in lowercase except for the first letter

File: galaxychop/core.py [hunk 2]
```diff
@@ -491,6 +491,12 @@ def jcirc(self, bin0=0.05, bin1=0.005):
...
+
+        if len(neg_star) == 0:
+            raise ValueError("All Stellar Energy values are positive")
```
Comment (ornemarioni) [inline galaxychop/core.py:498]: Same as above

File: galaxychop/core.py [hunk 3]
```diff
@@ -491,6 +491,12 @@ def jcirc(self, bin0=0.05, bin1=0.005):
...
+
         # Remove the particles with E = -inf.
         (fin,) = np.where(E_tot[neg] != -np.inf)
```
Comment (ornemarioni) [inline galaxychop/core.py:501]: Here you can implement this function: `np.isinf` or `~np.isinf` for the opossite function.
Check doc https://numpy.org/doc/stable/reference/generated/numpy.isinf.html

**Review by ornemarioni — COMMENTED** [review]
Please check the comments

**Pull request closed without merging**